frontend/apps/cloudstorage/dropbox.lua

```diff
@@ -0,0 +1,190 @@
+        "3. Now go on with the configuration, choosing the app permissions and access restrictions to your DropBox folder.\n"..
+        "4. Enter the >>App Name<< that you prefer (e.g. KOReader).\n"..
+        "5. Now, click on the >>Create App<< button.\n" ..
+        "6. When your new App is successfully created, please click on the Generate button.\n"..
+        "7. Under the 'Generated access token' section, then enter code in Dropbox token field."
+    local hint_top = _("Your Dropbox name")
+    local text_top = ""
+    local hint_bottom = _("Dropbox token\n\n\n\n ")
+    local text_bottom = ""
+    local title
+    local text_button_right = _("Add")
+    if item then
+        title = _("Edit Dropbox account")
+        text_button_right = _("Apply")
+        text_top = item.text
+        text_bottom = item.password
+    else
+        title = _("Add Dropbox account")
+    end
+    self.settings_dialog = MultiInputDialog:new {
+        title = title,
+        fields = {
+            {
+                text = text_top,
+                hint = hint_top ,
+            },
+            {
+                text = text_bottom,
+                hint = hint_bottom,
+                scroll = false,
+            },
+        },
+        buttons = {
+            {
+                {
+                    text = _("Cancel"),
+                    callback = function()
+                        self.settings_dialog:onClose()
+                        UIManager:close(self.settings_dialog)
+                    end
+                },
+                {
+                    text = _("Info"),
+                    callback = function()
+                        UIManager:show(InfoMessage:new{text = text_info })
+                    end
+                },
+                {
+                    text = text_button_right,
+                    callback = function()
+                        local fields = MultiInputDialog:getFields()
+                        if fields[1] ~= "" and fields[2] ~= "" then
+                            if item then
+                                self:saveEditedSettingsDropbox(fields, item, callback_refresh)
+                            else
+                                self:saveSettingsDropbox(fields, callback_refresh)
+                            end
+                            self.settings_dialog:onClose()
+                            UIManager:close(self.settings_dialog)
+                        else
+                            UIManager:show(InfoMessage:new{text = "Please fill in all fields." })
+                        end
+                    end
+                },
+            },
+        },
+        width = Screen:getWidth() * 0.95,
+        height = Screen:getHeight() * 0.2,
+        input_type = "text",
+    }
+    self.settings_dialog:onShowKeyboard()
+    UIManager:show(self.settings_dialog)
+end
+
+function DropBox:saveSettingsDropbox(fields, callback_refresh)
```

Comment:
Since it's already a method of the DropBox module, we can just name this method as `DropBox:saveSettings`. Same for other methods.